src/YesSql.Core/Services/DefaultQuery.cs

```diff
@@ -1046,7 +1046,7 @@ IQuery<T> IQuery<T>.Skip(int count)
             {
                 if (!_query._queryState._sqlBuilder.HasOrder)
                 {
-                    _query._queryState._sqlBuilder.OrderBy(_query._dialect.QuoteForColumnName("Id"));
+                    _query._queryState._sqlBuilder.OrderBy(_query._queryState._sqlBuilder.FormatColumn(_query._queryState._documentTable, "Id"));
```

Comment:
Added the table name to these Order By Id lines, to resolve "Ambiguous column " errors when the order by gets added to the Select.
Honestly I'm surprised this worked before, if it's ambiguous in the Select, it should be ambiguous in the OrderBy too right?

But now it's causing an error in the Postgre test saying:
Column "tpDocument.Id" does not exist. But that doesn't make sense, that column definitely exists.
I'm currently unable to test Postgre locally, so any advice would be helpful

Reply:
Got it, Postgre is case sensitive.  I didn't know that.